modules/sdk-coin-ada/src/lib/stakingActivateBuilder.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseKey, TransactionType } from '@bitgo/sdk-core';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { TransactionBuilder } from './transactionBuilder';
+import { Transaction } from './transaction';
+import * as CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
+
+export class StakingActivateBuilder extends TransactionBuilder {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this._type = TransactionType.StakingActivate;
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.StakingActivate;
+  }
+
+  /** @inheritdoc */
+  initBuilder(tx: Transaction): void {
+    super.initBuilder(tx);
+  }
+
+  stakingCredential(stakingPublicKey: string, poolHash: string): this {
```

Comment:
Can we get the jsdocs for this function?

Reply:
Oh man...forgive me, what does jsdocs refer to here?

Reply:
Not a problem at all. Here is an example: https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-coin-ada/src/ada.ts#L77
Here is the "official" source: https://jsdoc.app/index.html

Reply:
That one is missing the types. Should be like `@param {string} variableName <Description>`

Reply:
nit: missing curly braces. Not a blocker but should be fixed in this or another PR.